jbpm-wb-integration/jbpm-wb-integration-client/src/main/java/org/jbpm/workbench/wi/client/editors/deployment/descriptor/items/ObjectItemPresenter.java

```diff
@@ -102,6 +117,22 @@ public void signalParameterAddedOrRemoved() {
         fireChangeEvent();
     }
 
+    public void openEditModal() {
+        ItemObjectModel itemObjectModel = new ItemObjectModel();
+        itemObjectModel.setParameters(model.getParameters());
+        itemObjectModel.setResolver(model.getResolver());
+        Consumer<String> removeConsumer = e -> {
+            super.remove();
+        };
+        Consumer<String> editConsumer = removeConsumer.andThen(v->{
+            itemObjectModel.setValue(v);
+            this.getListPresenter().add(itemObjectModel);
+            fireChangeEvent();
+        });
+        singleValueModal.show(editConsumer);
+        singleValueModal.getView().setValue(model.getValue());
```

Comment:
same comments as the other presenter, avoid calling view directly and combine consumers.